docs/setup.md

```diff
@@ -10,9 +10,9 @@
 ## Prerequisites 
 The **MITRE ATT&CK&reg; Enrichment Framework** solution pack depends on the following solution packs.
 
-| Solution Pack Name | Purpose                                |
-|:-------------------|:---------------------------------------|
-| SOAR Framework     | Required for Incident Response modules |
+| Solution Pack Name | Version  | Purpose                                |
```

Comment:
I don't think we have a dependency of 2.1.0

Reply:
Confirmed with Ankit. SFSP 2.1 required